website/content/docs/nia/cli/index.mdx

```diff
@@ -11,6 +11,8 @@ Consul-Terraform-Sync (CTS) is controlled via an easy to use command-line interf
 
 ## Daemon
 
+~> Running daemon mode without a command is deprecated in CTS 0.6.0 and will be removed at a much later date in a major release. For information on the preferred way to run daemon mode see the [`start` command docs](/docs/nia/cli/start)
```

Comment:
@devarshishah3 can you please review the phrasing here?

Reply:
Looks good!